lib/rouge/lexers/bbcbasic.rb

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class BBCBASIC < RegexLexer
+      title "BBCBASIC"
+      desc "BBC BASIC syntax"
+      tag 'bbcbasic'
+      filenames '*,fd1'
```

Comment:
Is the comma meant to be a period/full stop?

Reply:
Probably not since you're specifically testing for that in the spec below. Man, that is _weird_.

Reply:
There's a long and complicated history behind that which I won't bore you with (unless you're actually interested!) but suffice it to say, yes it's a comma followed by 3 hex digits, acting as though it were a file extension.

Reply:
I'm always curious for obscure programming trivia but I better get through the rest of this review before I have to get off this train!